core/src/androidTest/java/de/danoeh/antennapod/core/tests/util/DateUtilsTest.java

```diff
@@ -122,4 +122,12 @@ public void testParseDateWithForCest() throws Exception {
         Date actual2 = DateUtils.parse("Sun, 29 Jan 2017 00:00:00 CET");
         assertEquals(expected2, actual2);
     }
+
+    public void testParseDateWithIncorrectWeekday() {
+        GregorianCalendar exp1 = new GregorianCalendar(2014, 9, 8, 9, 00, 00);
```

Comment:
Intellij warns that `00` is octal. Shouldn't be a problem but maybe we can avoid the warnings